tchannel/zipkin/tracers.py

```diff
@@ -40,13 +40,17 @@
 
 from .formatters import json_formatter
 from .formatters import thrift_formatter
+from .formatters import i64_to_base64
 from .thrift import TCollector
 from .thrift import constants
+from ..thrift import client_for
 
 log = logging.getLogger('zipkin_tracing')
 
 zipkin_log = logging.getLogger('zipkin')
 
+TCollectorClient = client_for('tcollector', TCollector)
```

Comment:
This introduces very tight Thrift and TCollector couping - are we absolutely sure we want that? cc @abhinav @blampe